app/views/curation_concerns/file_sets/_single_use_link_rows.html.erb

```diff
@@ -0,0 +1,18 @@
+<% single_use_links.each_with_index do |link, index| %>
+  <% type = link.path =~ /downloads/ ? 'Download' : 'Show' %>
+  <% url_helper = link.path =~ /downloads/ ? 'download_single_use_link_url' : 'show_single_use_link_url' %>
+  <tr>
+    <td><%= type %></td>
+    <td><%= link.downloadKey %></td>
```

Comment:
I'm `+0` on displaying the download key to users. Not sure it's meaningful for many of them. Certainly doesn't *hurt* to show it, so I won't complain too much about this.